recordedit/form.controller.js

```diff
@@ -145,14 +148,15 @@
                 // NOTE: This case is for a pseudo-failure case
                 // When multiple rows are updated and a smaller set is returned, the user doesn't have permission to update those rows based on row-level security
                 if (failedPage !== null) {
+                    var failedReference = failedPage.reference.contextualize.entryCompact;
```

Comment:
The page that is used here is created by create/update function. This means that it already is using the structure of reference and changing the context afterward doesn't have any effect.

So you should change the context in ermrestjs instead.

Reply:
for create:
https://github.com/informatics-isi-edu/ermrestjs/blob/master/js/reference.js#L1251
for edit:
https://github.com/informatics-isi-edu/ermrestjs/blob/master/js/reference.js#L1842